luxonis_train/attached_modules/losses/ocr_decoding_loss.py

```diff
@@ -0,0 +1,49 @@
+import torch
+from torch import Tensor, nn
+
+from .base_loss import BaseLoss
+
+
+class FocalCTC(BaseLoss[Tensor, Tensor]):
```

Comment:
Add class attribute `supported_labels = [LabelType.TEXT`]

Reply:
Also the metrics and losses will probably require custom `prepare` method that takes the packed output of the head and labels and prepares argument to be passed to `forward` or `update`.